src/commands/daily.ts

```
@@ -25,7 +25,7 @@ export const dailyCommand = define({
 		const dailyData = await loadDailyUsageData({
 			since: ctx.values.since,
```

ESLint

[error] Unsafe assignment of an `any` value.

(ts/no-unsafe-assignment)